resources/js/assets/lang/pl.json

```
@@ -0,0 +1,80 @@
+{
+    "GUI": {
+        "Prices" : "Ceny",
+        "Find_a_ride" : "Znajdź przejazd",
+        "Rules" : "Zasady",
+        "Loading" : "Ładowanie",
+        "Admin_panel" : "Panel administratora",
+        "Close_menu" : "Zamknij menu",
+        "Open_menu" : "Otwórz menu",
+        "Search_city" : "Szukaj miasta",
+        "Search_country" : "Szukaj kraju",
+    },
+    "Auth": {
+        "Register" : "Zarejestruj",
+        "Name" : "Nazwa",
+        "Email" : "E-mail",
+        "Password" : "Hasło",
+        "Log_in" : "Zaloguj",
+        "Log_out" : "Wyloguj",
+        "Sign_up" : "Zarejestruj się",
+        "Confirm_password" : "Potwierdź hasło",
+    },
+    "Prompt": {
+        "No_account" : "Nie masz konta?",
+        "Already_have_account" : "Masz już konto?",
+        "Looking_for_an_escooter" : "Szukasz hulajnogi elektrycznej?",
+        "Try_it_out" : "Wypróbuj teraz!",
+        "Create_account" : "Stwórz konto",
+        "Login_to_add_to_favorite" : "Zaloguj się, aby dodać do ulubionych",
+        "Sorry_we_couldnt_cities" : "Przepraszamy, ale nie mogliśmy znaleźć żadnych miast",
+        "Sorry_we_couldnt_countries" : "Przepraszamy, ale nie mogliśmy znaleźć żadnych krajów",
+    },
+    "Info": {
+        "advert_1" : "Jest wielu dostawców hulajnóg elektrycznych, więc czasami możesz być zdezorientowany, których aplikacji powinieneś używać podczas swoich podróży.",
```

```suggestion
        "advert_1" : "Jest wielu dostawców hulajnóg elektrycznych, więc czasami możesz być zdezorientowany/a, których aplikacji powinieneś/aś używać podczas swoich podróży.",
```

Or rewrite this sentence somehow to avoid gender distinctions

---

fixed